roles/openshift_logging_elasticsearch/tasks/main.yaml

```diff
@@ -289,7 +283,7 @@
     es_memory_limit: "{{ openshift_logging_elasticsearch_memory_limit }}"
     es_node_selector: "{{ openshift_logging_elasticsearch_nodeselector | default({}) }}"
     es_storage_groups: "{{ openshift_logging_elasticsearch_storage_group | default([]) }}"
-    es_container_security_context: "_es_containers | default({})| attr('elasticsearch')|default({})|attr('securityContext')"
+    es_container_security_context: "{{ _es_containers.elasticsearch.securityContext if _es_containers is defined and 'elasticsearch' in _es_containers and 'securityContext' in _es_containers.elasticsearch else None }}"
```

Comment:
jinja2 filter `attr()` does something else than get a value by its key from dict. It is well described here - https://github.com/ansible/ansible/issues/24065